### PR TITLE
Add OMGServ and fix Noez doc

### PR DESCRIPTION
**Pull request by Incendit**
OMGServ Alexa ranking :
firefox_2020-08-25_15-13-50

The documentation is only available in French but there some images, - fr regions and exception line added because of this.

Noez documentation link was pointing on Q&A homepage, now it's 2FA article ⚠️ still in Deutsch.

### CODE REVIEW

File: _data/hosting.yml [hunk 1]
```diff
@@ -425,6 +425,17 @@ websites:
...
     regions:
       - se
+      
+  - name: OMGServ
+    url: https://www.omgserv.com
+    img: omgserv.png
+    tfa:
+      - totp
+    doc: https://www.omgserv.com/fr/faq-site/comment_activer_la_double_authentification-103/
+    exception: "Documentation only available in French, but illustrated"
```
Comment (Carlgo11) [inline _data/hosting.yml]: With the region tag I think it's unnecessary to include this exception note.

Thanks for being so thorough though! 🙂

Comment (Incendit) [reply _data/hosting.yml]: You're right. I've deleted the exception line :)

File: _data/hosting.yml [hunk 2]
```diff
@@ -412,7 +412,7 @@ websites:
     img: noez.png
     tfa:
       - totp
-    doc: https://noez.de/de/faq/?r=199
+    doc: https://noez.de/en/faq/post/199/Wie-aktiviere-ich-die-2FA-(Zwei-Faktor-Authentisierung)-f%C3%BCr-meinen-Account/
```
Comment (phallobst) [inline _data/hosting.yml]:
```suggestion
    doc: https://noez.de/en/faq/post/199/
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Comment by phallobst** [comment]
TinyPNG was able to reduce the icon file size:
omgserv

**Review by Carlgo11 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**